webapp-mgmt/cas-management-webapp-configuration/build.gradle

```diff
@@ -0,0 +1,39 @@
+description = "Apereo CAS Management Web Application Configuration"
+
+repositories {
+    mavenLocal()
+    jcenter()
+    maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
+    maven { url 'https://build.shibboleth.net/nexus/content/repositories/releases/' }
+}
+
+dependencies {
+    implementation libraries.springcloudconfigclient
+
+    optional libraries.springbootconfigmetadata
+
+    implementation "org.apereo.cas:cas-server-core-api-configuration:$casVersion"
+    implementation "org.apereo.cas:cas-server-core-util:$casVersion"
+    implementation "org.apereo.cas:cas-server-core-configuration-api:$casVersion"
+
+}
+
+task generateConfigurationMetadata(type: JavaExec) {
+    doFirst {
+        println "\nRunning CAS Spring configuration metadata processor in addition to the native " +
```

Comment:
I'll take a look at this bit later. Fine as is. @hdeadman indicated there may be an issue with the generator not honoring parent classes. If I get that fixed, I'll run over to the mgmt app and see if I can get it to also generate metadata correctly.